.eslintrc.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
 	},
 	env: {
 		browser: true,
+    "jest": true
```

Comment:
This doesn't seem to be indented properly, are you using the prettier plugin in your IDE? That would normally fix that automatically, I think.